app/views/clients/hud_assessments/show.haml

```diff
@@ -11,7 +11,7 @@
       %h3 Assessment Details
       %dl
         %dt Project
-        %dd= GrdaWarehouse::Hud::Project.confidentialize(name: @assessment.enrollment.project&.name)
+        %dd= @assessment.enrollment.project&.name(current_user)
```

Comment:
We'll need to check that the controller `.preloads(enrollment: :project)`

Reply:
it does 👍